packages/forms/stories/story-types.ts

```diff
@@ -131,6 +170,83 @@ export const FIELDSET_ARGS_TYPES = {
   }
 };
 
+export const FILE_UPLOAD_ARG_TYPES = {
+  isHidden: {
+    name: 'Hidden label'
+  },
+  showHint: {
+    name: 'Hint'
+  },
+  showMessage: {
+    name: 'Message'
+  },
+  validation: {
+    name: 'Validation',
+    control: {
+      type: 'radio',
+      options: ['success', 'warning', 'error']
+    }
+  },
+  isDragging: {
+    table: {
+      disable: true
+    }
+  },
+  type: {
+    control: {
+      type: 'select',
+      options: [
+        'pdf',
+        'zip',
+        'image',
+        'document',
+        'spreadsheet',
+        'presentation',
+        'generic',
+        undefined
+      ]
+    }
+  }
+};
+
+export const FILE_LIST_ARG_TYPES = {
+  focusInset: {
+    name: 'Inset File box-shadow'
+  },
+  includeProgress: {
+    name: 'Include Progress'
+  },
+  type: {
+    control: {
+      type: 'select',
+      options: [
+        'pdf',
+        'zip',
+        'image',
+        'document',
+        'spreadsheet',
+        'presentation',
+        'generic',
+        undefined
+      ]
+    }
+  },
+  remove: {
+    name: 'Include Remove',
+    control: {
+      type: 'radio',
+      options: ['File.Close', 'File.Delete']
+    }
+  },
+  validation: {
+    name: 'Validation',
+    control: {
+      type: 'radio',
```

Comment:
Not a blocker but curious if a select knob with the option "none" would be a better alternative. Currently, once a user selects a validation, they can no longer "unset" it and cannot see different types of file icons.

Reply:
I had assumed Storybook's reset controls button was sufficient. But I see what you're saying about the ease of exploring file types with validation set.